src/datastore/labelsgroup.cpp

```diff
@@ -49,6 +49,22 @@ void DsLabelsGroup::clear()
 	this->indexLimit = 0;
 }
 
+int DsLabelsGroup::removeGroup(DsLabelsGroup& otherGroup)
+{
+	// This could be made faster by working directly with data blocks
+	int numberRemoved = 0;
+	DsLabelIndex index = -1; // DS_LABEL_INDEX_INVALID
```

Comment:
Is there a reason for using an integer here and not the enumeration?  Maybe it is better to cast it once from a `DS_LABEL_INDEX_INVALID `?